packs/data/feats.db/sly-striker.json

```diff
@@ -30,15 +30,23 @@
         },
         "rules": [
             {
-                "damageType": "precision",
+                "category": "precision",
                 "dieSize": "d6",
                 "key": "DamageDice",
                 "predicate": [
+                    "class:rogue",
                     {
                         "or": [
-                            "finesse",
-                            "agile",
-                            "ranged"
+                            "weapon:trait:agile",
+                            "weapon:trait:finesse",
+                            {
+                                "and": [
+                                    "weapon:ranged",
+                                    {
+                                        "not": "weapon:thrown-melee"
```

Comment:
```suggestion
                                        "not": ["weapon:thrown-melee"]
```

Reply:
`not` takes a single string (the original is correct)